### ISSUE TITLE
Move BasicEntityPersister to OMG\Persister

### ISSUE THREAD

**Issue by Nyholm** [graphaware/neo4j-php-ogm#92]
We have two namespaces `GraphAware\Neo4j\OGM\Persister` and `GraphAware\Neo4j\OGM\Persisters`. Generally namespaces should be in singular. 
I suggest moving the the `BasicEntityPersister` to `GraphAware\Neo4j\OGM\Persister`

**Comment by ikwattro** [comment]
I agree the names are confusing. `Persister` was a personally chosen namespace before the Doctrine like architecture. I'm happy to mark it for RC2.

**Comment by Nyholm** [comment]
👍
